tests/data/version/embree/recipe.yaml

```diff
@@ -0,0 +1,47 @@
+context:
```

Comment:
```suggestion
# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
context:
```

Reply:
We don't need this anymore because the recipe is in the schema store :)
<img width="483" alt="Screenshot 2024-08-07 at 09 18 48" src="https://github.com/user-attachments/assets/900b76a2-9ffa-463e-933e-949434d8672f">